### DISCUSSION TITLE
Your site is published at differentusername.github.io

### DISCUSSION THREAD

**Discussion by AhmedKAwwad** [community/community#22036] [Other Feature Feedback, Questions, & Ideas]
<p>I made my repo <a href="http://Ahmed3wwad.github.io" rel="noopener nofollow ugc">Ahmed3wwad.github.io</a><br>
but when i enable github pages it gives an old repo page i already deleted as shown<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/6/3636d0ab5f5c47dcca9c0d8478c87c8ecd649af4.png" data-download-href="/uploads/short-url/7JBdU0bHKSJvCm5ocDa1OD1nMYQ.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/23080326/181068388-65a74eb0-e9cc-4330-8d61-c3ae351b08f3.png" alt="image" data-base62-sha1="7JBdU0bHKSJvCm5ocDa1OD1nMYQ" width="690" height="173" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/6/3636d0ab5f5c47dcca9c0d8478c87c8ecd649af4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">821×207 8.59 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div><br>
i tried to disable pages and delete repo and re create everything but hopeless.
<p>How to disable that old name forever i want to use a new one as my current repo</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Your <code>github.io</code> subdomain will always be your username, so for you it’s <code>ahmedkawwad.github.io</code>. To publish a user page your repository name must exactly match that domain. The only way to change that domain is to change your username. Or you could use a custom domain.</p>
<p>All other repositories with Pages enabled will be published in a subdirectory under that domain, like you see here.</p>

**Comment by AhmedKAwwad** [comment]
<p>so it’s not depending on the repo name , oh sounds i miss understand things here , thanks for clarification.</p>

**Comment by jfullstackdev** [comment]
<p>or maybe you can create an org for a specific name, like <code>AhmedKAwwad-tutorial</code>, this way, when you enable that GitHub Page, it’s entirely a new name, the name that you want, plus, you can create more, so you don’t need to change your name when you want another one for the GitHub Page</p>

**Comment by AhmedKAwwad** [comment]
<p>I didn’t get the way , you mean create a new account with another name like <code>AhmedKAwwad-tutorial</code> , or you mean i create new repo ?<br>
incase you mean new account , i got it okay .<br>
incase you mean new repo , then it will be published on <code>AhmedKAwwad.github.io/AhmedKAwwad-tutorial/</code>.<br>
That’s why i am stuck with the user name at the published url.</p>

**Comment by jfullstackdev** [comment]
<p>for example, I created a GitHub Org,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/5/7559c3be3caf47f203ef5355facca8cf0a82b52d.png" data-download-href="/uploads/short-url/gK8boKjX0RINb51dIbQixhKUJ53.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068397-8475b3b8-d6a9-4157-8fcb-1ae798ccdf5c.png" alt="image" data-base62-sha1="gK8boKjX0RINb51dIbQixhKUJ53" width="690" height="410" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/5/7559c3be3caf47f203ef5355facca8cf0a82b52d_2_690x410.png, https://aws1.discourse-cdn.com/github/original/3X/7/5/7559c3be3caf47f203ef5355facca8cf0a82b52d.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/5/7559c3be3caf47f203ef5355facca8cf0a82b52d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/5/7559c3be3caf47f203ef5355facca8cf0a82b52d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">724×431 27.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>if I will be creating a GitHub Page for that Org, it will be as if it’s another user,</p>
<p>so that will be</p>
<p><code>xkcph2017.github.io</code></p>
<p>so, in case you want another separate GitHub Page, that can be done through GitHub Org, without creating another user</p>

**Reply by ftth05** [reply]
Is this still valid?

**Comment by AhmedKAwwad** [comment]
<aside class="quote no-group" data-username="jdevstatic" data-post="6" data-topic="239839">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>user</p>
</blockquote>
</aside>
<p>Oh nice trick , thanks for this one :grin:</p>
